Parameter to disable extrapolation in sharp IB solver (#835)
lethe-cfd/lethe: doc/source/parameters/sharp-immersed-boundary-solver/sharp-immersed-boundary-solver.rst

    .. tip::
	    A good starting value is twice the average aspect ratio of the elements in the mesh multiplied by the order of the underlying FEM scheme.

    * The ``enable extrapolation`` parameter controls if extrapolation is used to impose the immersed boundary condition. For debugging purposes, this parameter can be set to ``false``; the particle velocity will then be imposed on velocity degrees of freedom of cells cut by the particle directly, which effectively amplifies the volume occupied by the solid.

    .. warning::

>     	Disabling the extrapolation is not recommended since it makes the Sharp-IB solver lose its high-order capabilities, akin to more basic IB methods.

Reviewer: ```suggestion
    	Disabling the extrapolation is not recommended since it makes the Sharp-IB solver first-order accurate in space.
```
